test/plots/cars-mpg.js

```diff
@@ -9,7 +9,8 @@ export default async function() {
     },
     y: {
       grid: true,
-      zero: true
+      zero: true,
+      axis: "both"
```

Comment:
It’d be better to have a new test that demonstrates a more appropriate use of *both* axis rather than tacking it on to some of the existing tests.